lib/services/fanout.go

```diff
@@ -33,12 +33,18 @@ type fanoutEntry struct {
 	watcher *fanoutWatcher
 }
 
+type kindSubKind struct {
```

Comment:
This is almost the same as `resourceKind`, but doesn't have `version`. I could use it instead of this type assuming `version` would always be empty, but it didn't seem right.

Maybe if we clean up [that version condition for KindAppServer](https://github.com/gravitational/teleport/blob/2f09551c7cbb95e40b3664ef89718fa6fa4597c7/lib/services/local/events.go#L108) and remove `version` from `resourceKind`, we won't need this extra type.